contracts/libraries/VersionUtils.sol

```diff
@@ -110,21 +110,18 @@ library VersionUtils {
     /**
      * @notice Used to packed the KYC data
      */
-    function packKYC(uint64 _a, uint64 _b, uint64 _c, uint8 _d, uint8 _e, uint8 _f) internal pure returns(uint256) {
-        return (uint256(_a) << 152) | (uint256(_b) << 88) | (uint256(_c) << 24) | (uint256(_d) << 16) | (uint256(_e) << 8) | uint256(_f);
+    function packKYC(uint64 _a, uint64 _b, uint64 _c, uint8 _d) internal pure returns(uint256) {
```

Comment:
Could we have a comment explaining the function here a bit for auditors / people reviewing the code etc.